django/researchdata/views.py

```diff
@@ -2,11 +2,25 @@
 from . import cwb_exec
 
 
-class TestView(TemplateView):
+class MonolingualCorporaView(TemplateView):
     """
-    Class-based view to show the test template
+    Class-based view to show the Monolingual Corpora template
     """
-    template_name = 'researchdata/test.html'
+    template_name = 'researchdata/monolingual-corpora.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        query_input = self.request.GET.get('query', '')
+        context['query_input'] = query_input
+        context['query_output'] = cwb_exec.query(context=query_input).split('\n')[0]
```

Comment:
Do we want to do this even if the query is empty? Also, this will only show the first line of output - is that the plan?

Reply:
This isn't 'real' code yet, just test code to get it returning _something_ as it's in ongoing development (this PR is just to set up pages). The two points you mentioned will be resolved in future PRs